src/Common/src/CoreLib/System/Numerics/GenerationConfig.ttinclude

```diff
@@ -30,6 +30,11 @@
         typeof(UInt32), typeof(UInt64)
     };
 
+    public static Type[] unsupportedTypes = new[]
```

Comment:
I don't think we need to put this list in this file.  It isn't shared with anything else, and can just be in the `GenericVectorTests.netcoreapp.tt` file.

Reply:
Done.